packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -414,7 +438,12 @@ class TextSpan extends InlineSpan {
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
 
-    properties.add(StringProperty('text', text, showName: false, defaultValue: null));
+    properties.add(StringProperty(
+      'text',
+      text,
```

Comment:
same here

Reply:
I'm missing it here as well. Where? :)

Reply:
I might be wrong, but do we need StringProperty in a new line?
```dart
properties.add(
    StringProperty(
        'text',
         ....
     )
);
```